Funcky.Analyzer/Funcky.Analyzer.Vsix/Funcky.Analyzer.Vsix.csproj

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+    <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+    <Import Sdk="Microsoft.Build.CentralPackageVersions" Project="Sdk.props" />
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <RootNamespace>Funcky.Analyzer.Vsix</RootNamespace>
+        <AssemblyName>Funcky.Analyzer.Vsix</AssemblyName>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <GeneratePkgDefFile>false</GeneratePkgDefFile>
+        <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
+        <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>
+        <IncludeDebugSymbolsInLocalVSIXDeployment>false</IncludeDebugSymbolsInLocalVSIXDeployment>
+        <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+        <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+        <VSSDKTargetPlatformRegRootSuffix>Roslyn</VSSDKTargetPlatformRegRootSuffix>
+    </PropertyGroup>
+
+    <ItemGroup Condition="'$(MSBuildRuntimeType)' == 'Full'">
+        <PackageReference Include="Microsoft.VSSDK.BuildTools" PrivateAssets="all" />
+    </ItemGroup>
+
+    <PropertyGroup>
+        <StartAction>Program</StartAction>
+        <StartProgram>$(DevEnvDir)devenv.exe</StartProgram>
+        <StartArguments>/rootsuffix $(VSSDKTargetPlatformRegRootSuffix)</StartArguments>
+    </PropertyGroup>
+
+    <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
+
+    <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="Exists('$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets')" />
+
+    <ItemGroup>
+        <ProjectReference Include="..\Funcky.Analyzer.CodeFixes\Funcky.Analyzer.CodeFixes.csproj" />
+        <ProjectReference Include="..\Funcky.Analyzer\Funcky.Analyzer.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <!-- https://github.com/dotnet/sdk/issues/433 -->
+        <ProjectReference Update="@(ProjectReference)" AdditionalProperties="TargetFramework=netstandard2.0" />
+
+        <!-- https://github.com/Microsoft/extendvs/issues/57 -->
```

Comment:
Dead link

Reply:
Autogenerated by template

Reply:
the issues on the project are deactivated, but the project still exists.